submissions/f3d0t/html-css-popup/index.html

```diff
@@ -0,0 +1,188 @@
+                                            </img>
+                                        </a>
+                                    </li>
+                                    <li class="popup__item">
+                                        <a class="popup__link" href="#">
+                                            <img alt="News" class="popup__link--img" src="img/news.png">
+                                                <span class="popup__link--text">
+                                                    News
+                                                </span>
+                                            </img>
+                                        </a>
+                                    </li>
+                                    <li class="popup__item">
+                                        <a class="popup__link" href="#">
+                                            <img alt="Search" class="popup__link--img" src="img/search.png">
+                                                <span class="popup__link--text">
+                                                    Search
+                                                </span>
+                                            </img>
+                                        </a>
+                                    </li>
+                                    <li class="popup__item">
+                                        <a class="popup__link" href="#">
+                                            <img alt="Translate" class="popup__link--img" src="img/tanslation.png">
+                                                <span class="popup__link--text">
+                                                    Translate
+                                                </span>
+                                            </img>
+                                        </a>
+                                    </li>
+                                    <li class="popup__item">
+                                        <a class="popup__link more__link" href="https://kottans.org/" target="_blank">
+                                            <span>
+                                                Even more from Kottans
+                                            </span>
+                                        </a>
+                                    </li>
+                                </ul>
+                            </input>
+                        </div>
+                    </input>
+                </li>
+                <li class="menu__item">
+                    <a class="menu__item--link--img" href="#">
+                        <img alt="Bell button" class="popup__button--img" src="img/bell.png">
+                        </img>
+                    </a>
+                </li>
+                <li class="menu__item">
+                    <a class="menu__item--link--img" href="#">
+                        <img alt="User avatar" class="popup__button--img avatar" src="img/apps.png">
```

Comment:
Can you play around icon images and decrease unnecessary sizes?
<img width="649" alt="Screenshot 2020-12-11 at 19 07 19" src="https://user-images.githubusercontent.com/42999342/101933078-58016200-3be4-11eb-8a52-22acb84ae7fc.png">